airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -512,7 +512,7 @@ private ConnectorSpecification getSpecFromDestinationDefinitionId(final UUID des
   private JobInfoRead createNewSchedulerCancellation(final Long id) throws IOException {
```

Comment:
name of this class is a bit hard for me to follow. had to go open the class to figure out what it was all about. would `createNewSchedulerCancellation ` => `submitCancellationToWorker` help?

Reply:
If you do change this probably worth doing the same for `createManualRun` => `submitManualSyncToWorker` and `resetConnectionWithNewScheduler` => `submitResetConnectionToWorker`

Reply:
Good suggestions. Done.